src/OSPSuite.UI/UIConstants.cs

```diff
@@ -1,12 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Drawing;
+using DevExpress.Skins;
 using OSPSuite.Assets;
 
 namespace OSPSuite.UI
 {
    public static class UIConstants
    {
+      public static class Colors
+      {
+         public static Color Disabled => skinColorFor("ReadOnly");
```

Comment:
maybe add a comment explaining why you are using a method get and not a property (so that it always gets the latest color from the theme)